rundeckapp/src/main/groovy/com/dtolabs/rundeck/server/plugins/RundeckEmbeddedPluginExtractor.groovy

```diff
@@ -40,11 +37,11 @@ class RundeckEmbeddedPluginExtractor implements ApplicationContextAware, Initial
 
     ApplicationContext applicationContext
     File pluginTargetDir
-    RundeckPluginRegistry rundeckPluginRegistry
```

Comment:
This is not used and also created a circular dependency with `RundeckPluginRegistry`.